packages/cli/internal/pkg/cli/util/timeout.go

```diff
@@ -0,0 +1,25 @@
+package util
+
+import (
+	"errors"
+	"time"
+)
+
+const (
+	timeoutError = "the deployment is taking longer than expected. Please review the stack in CloudFormation"
```

Comment:
We should retrieve the relevant information for the user since we know the stack name and the region it is being deployed to instead of just providing a generic message